integration_tests/macros/my_metrics.sql

```diff
@@ -22,4 +22,48 @@
 {% macro re_data_metric_distinct_count(column_name) %}
     count(distinct( {{column_name}} ))
 
-{% endmacro %}
\ No newline at end of file
+{% endmacro %}
+
+{% macro re_data_metric_regexp_test(column_name, config) %}
+    {{ regexp_test(column_name, config) }}
+{% endmacro %}
+
+{% macro regexp_test(column_name, config) %}
+    {{ adapter.dispatch('regexp_test', project_name)(column_name, config) }}
+{% endmacro %}
+
+{% macro default__regexp_test(column_name, config) %}
+    {% set pattern = config.get('regexp') %}
+    coalesce(
+        sum(
+            case when {{column_name}} ~ '{{pattern}}'
```

Comment:
you can probably only copy part under case? since rest is the same?

Reply:
👍🏽